deploy/operator.yaml

```diff
@@ -13,13 +13,23 @@ spec:
         name: rhmi-operator
     spec:
       serviceAccountName: rhmi-operator
+      volumes:
+      - name: webhook-certs
```

Comment:
I am not as familiar with the cluster certs and the usage within pods. Is there any side affects of adding this in?

Reply:
@davidffrench 
1. Yes, I'm planning to define those methods so that the new CR implements the `Validator` interface once the PR for the CRD is merged.
2. That's a good question. I'm not too familiar with OLM yet so I'm not sure about that, but the service must be created before the operator pod, as the pod mounts the certificates from the secret created after the service is annotated.

Reply:
It seems to be supported by Openshift 4. I took this from [the documentation](https://docs.openshift.com/container-platform/4.1/authentication/certificates/service-serving-certificate.html) and it's working very well on the clusters I've tested it with. The use case matches 100% what I needed, as the webhook server from controller-runtime is very restrictive, all you can configure is the directory where the certificates are, so mounting it as a volume does the job well

Reply:
Perfect, thanks Sergio